### DISCUSSION TITLE
NEMO RANS solver?

### DISCUSSION THREAD

**Discussion by eskimmel** [su2code/SU2#1614] [Q&A] [answered]
Hi all,

It was mentioned in this [thread](https://www.cfd-online.com/Forums/su2/228250-su2-v7-0-5-hypersonic.html) that a NEMO RANS solver was in development a bit over a year ago. I am trying to model a compression ramp in hypersonic flow with DES, and I would like to include nonequilibrium effects in the boundary layer calculations. Since a NEMO RANS solver would be ideal for the calculations I am trying to perform, I was wondering how its development is coming along and if we should be expecting its availability anytime soon?

**Comment by WallyMaier** [comment, marked as answer]
Hi @eskimmel,

The RANS capabilities are nearly complete and are going through some verification and validation cases.  I would expect the capability to be ready in about a month.

**Reply by eskimmel** [reply]
Great, thanks!
